src/WebJobs.Script.Scaling/AzureEnvironments.cs

```diff
@@ -0,0 +1,200 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Win32;
+
+namespace Microsoft.Azure.WebJobs.Script.Scaling
+{
+    public static class AzureEnvironments
```

Comment:
This name seems off. Rename to something like AzureEnvironmentConstants or AzureConstants something?